### ISSUE TITLE
API Credentials not pre-filling or being acknowledged when manually entering

### ISSUE THREAD

**Issue by zeldridge** [EthanArbuckle/Apollo-CustomApiCredentials#27]
I sideloaded this app with sideloadly and injected both:

- 1.0.0-1_iphoneos-arm-rootfull.deb
- ApolloCustomApiCredentials_1.0.0-1.dylib

But the banner up the top doesn't change to "select "create app"" and prefill the fields. I tried manually typing the api info, it doesn't do anything to the app either.

How can I solve this? I am not JB if that helps

**Comment by nicko88** [comment]
Same here.

After I log in, it goes to the developer page and there is only a create new app button on the bottom that I can click, but when I click it, it opens the create new api fields and they are all blank, nothing is automatically populated.

If I fill in stuff and create then it doesn't seem to load anything into the apollo app at all.

Not sure what I could be doing wrong.  It doesn't work like the animation in the readme.

**Comment by EthanArbuckle** [comment]
Looking into this. Does your account already have an app / API credential created?

**Comment by FogDogg** [comment]
Same here. I’m on iOS 17 DB2 and using sideloady (with only cydia checked and app extensions removed). 

Tried 2 different accounts - one with a previous api key and a fresh account with no prior api keys.

**Comment by nicko88** [comment]
I do have a previous api key set up.

I used the previous version of your code that I had to compile myself.

But I wanted to switch to this update so that I can have a randomized user-agent.

**Comment by EthanArbuckle** [comment]
I was able to reproduce this with an account that has an existing API credential created. I deleted the existing app (in a desktop browser) and then re-opened Apollo, and the tweak started working correctly.

I'm working on a fix, and i'll also add some type of fallback in case the automation breaks again in the future (It won't surprise me if reddit tries to break things intentionally at some point)

Apologies for the issues!
